tests/upload.int.test.ts

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import 'mocha';
+import { Client } from '../src/client';
+import * as tmp from 'tmp';
+import * as fs from 'fs';
+import * as os from 'os';
+import * as path from 'path';
+import { Storage } from '@google-cloud/storage';
+import pMap from 'p-map';
+import {
+  EXAMPLE_FILE,
+  EXAMPLE_DIR,
+  FILES_IN_DIR,
+  EXAMPLE_PREFIX,
+  FILES_IN_DIR_WITHOUT_PARENT_DIR,
+  TXT_FILES_IN_DIR,
+  TXT_FILES_IN_DIR_WITHOUT_PARENT_DIR,
+  TXT_FILES_IN_TOP_DIR,
+} from './constants.test';
+
+const storage = new Storage({
+  projectId: process.env.UPLOAD_CLOUD_STORAGE_TEST_PROJECT,
```

Comment:
I don't see this env var passed into the test via the workflow?

Reply:
oops added it. Some tests were silently being skipped.